cypress/support/cypress-commands/commands.js

```diff
@@ -1388,7 +1388,6 @@ Cypress.Commands.add(
     appId = UTILS.fetchAppIdentifierFromEnv(appId);
     const context = {};
     const expectingError = errorContent ? true : false;
-    contentObject = contentObject ? contentObject : CONSTANTS.NULL_RESPONSE;
     method = method.includes('_') ? method.split('_')[1] : method;
```

Comment:
why did we remove this ? isnt this default validation added for setters ?

Reply:
this condition is already present in validation glues